ds3-sdk/src/test/java/com/spectralogic/ds3client/Ds3ClientBuilder_Test.java

```diff
@@ -31,6 +32,27 @@ public void createBasicClient() throws Exception {
     }
 
     @Test
+    public void createBasicClientFromENV() throws Exception {
+        try {
+            final Ds3ClientBuilder builder = Ds3ClientBuilder.fromEnv();
```

Comment:
but those test should be generic, maybe i'll run this test when the env vars are not setup in my system, why should i see a fail test?